validation/td-validation.ttl

```diff
@@ -1082,7 +1094,7 @@
                 <td>
                     0..*
                 </td>
-                <td>Point to a software implementation of the TD.</td>
+                <td>Point to the web app manifest of a web application which provides a user interface with which a user can interact with the Thing (also see [[APPMANIFEST]]).</td>
```

Comment:
See previous comment. This introduces a new specification, which is a working draft.

Reply:
is reverted. see comment above